router/forwarder.go

```diff
@@ -38,11 +38,11 @@ func (conn *LocalConnection) ensureForwarders() error {
 	}
 
 	forwarder := NewForwarder(conn, encryptor, udpSender, DefaultPMTU)
-	forwarderDF := NewForwarder(conn, encryptorDF, udpSenderDF, DefaultPMTU)
+	forwarderDF := NewForwarderDF(conn, encryptorDF, udpSenderDF, DefaultPMTU)
 	effectivePMTU := forwarderDF.unverifiedPMTU
 	//NB: only forwarderDF can ever encounter EMSGSIZE errors, and
 	//thus perform PMTU verification
```

Comment:
Yeah.  Indeed, it seems redundant now, as it ought to be clearer from the code that forwarder and forwarderDF are different and how they are different.